src/defaultsettings.cpp

```diff
@@ -56,12 +59,11 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("keymap_toggle_hud", "KEY_F1");
 	settings->setDefault("keymap_toggle_chat", "KEY_F2");
 	settings->setDefault("keymap_toggle_force_fog_off", "KEY_F3");
-	settings->setDefault("keymap_toggle_update_camera",
+	settings->setDefault("keymap_toggle_update_camera", ""
```

Comment:
Please don't do this. I think it's better to have more typing and have both entire lines in the body of #if and #else